front/src/routes/integration/all/netatmo/discover-device/DiscoverTab.jsx

```diff
@@ -0,0 +1,70 @@
+import { Component } from 'preact';
+import { Text } from 'preact-i18n';
+import cx from 'classnames';
+
+import EmptyState from '../EmptyState';
+import style from '../style.css';
+import CheckMqttPanel from '../../mqtt/commons/CheckMqttPanel';
+import TasmotaDeviceBox from '../TasmotaDeviceBox';
```

Comment:
Looks like a fast copy/paste :) 
May cause confusions, we are not in Tasmota integration here

Reply:
Hi !! 
Thank you @atrovato for your review.  Well indeed all these points are not normal, it was supposed to be modified and corrected on an old merge.  I think there was a problem between our different merges. The files are fine with me and have no errors.  Sorry for the time wasted.  I correct and merge / push again tomorrow.

Reply:
Ok sorry, files deleted here : https://github.com/GladysAssistant/Gladys/pull/1015/commits/cd9f2252a960c4fbab8cb3212590c55bc8603496